internal/ota/boadpids.go

```diff
@@ -0,0 +1,68 @@
+// This file is part of arduino-cloud-cli.
```

Comment:
I believe there is a typo in the file name

Reply:
I'v enot understood this. This header is the same in all files